internal/graph/dot.go

```diff
@@ -31,50 +31,57 @@ func (ctx *Dot) Label(p *pkggraph.Node) string {
 	return labelText.String()
 }
 
-func (ctx *Dot) ClusterLabel(tree *pkggraph.Tree, parentPrinted bool) string {
-	var suffix = ""
-	if parentPrinted && tree.Parent != nil && tree.Parent.Path != "" {
-		suffix = "./" + strings.TrimPrefix(tree.Path, tree.Parent.Path+"/")
+func (ctx *Dot) ModuleLabel(mod *pkggraph.Module) string {
```

Comment:
Wondering whether customizing this would be helpful... probably not significantly.

Reply:
I thought of that too, and it would make sense since the package labels can be customized with a template. But I also wasn't sure how much people would want to use it. It wouldn't be hard to add, but maybe wait until someone has a use case for it.

Reply:
Yeah, let's wait for a request.